packages/core/src/routes/well-known.ts

```diff
@@ -24,17 +24,21 @@ export default function wellKnownRoutes<T extends AnonymousRouter>(
   } = queries;
 
   if (tenantId === adminTenantId) {
-    router.get('/.well-known/endpoints/:tenantId', async (ctx, next) => {
-      if (!ctx.params.tenantId) {
-        throw new RequestError('request.invalid_input');
-      }
+    router.get(
+      '/.well-known/endpoints/:tenantId',
+      koaGuard({ response: z.object({ user: z.string().url() }), status: 200 }),
+      async (ctx, next) => {
+        if (!ctx.params.tenantId) {
+          throw new RequestError('request.invalid_input');
+        }
```

Comment:
should replace this with `params` guard, otherwise it will respond with 500 if you don't specify `400` in the `status` guard

Reply:
hmmm, good point!